lib/ansible/modules/source_control/github_release.py

```diff
@@ -172,14 +178,17 @@ def main():
 
     # login to github
     try:
-        if user and password:
+        if password:
```

Comment:
`user` is still a `required=True` option to the module so this doesn't change the behavior

Reply:
Thanks for the review!

Indeed, `user` is always required because it's need to build the full repository URL (so for the Ansible repository, you would set `user=ansible` and `repo=ansible` which gives you `https://github.com/ansible/ansible` as a URL). I removed `user` from this condition precisely because it's always `True` anyway.

The change that really modifies the behavior is the [removal of line 152](https://github.com/ansible/ansible/pull/47817/files#diff-7b3549c33449c0e62b49d2ff81df2c7dL152).